sdk/communication/communication-phone-numbers/recordings/browsers/phonenumbersclient__get_phone_number/recording_can_get_a_purchased_phone_number.json

```diff
@@ -12,18 +12,19 @@
    "responseHeaders": {
     "api-supported-versions": "2021-03-07, 2022-01-11-preview2",
     "content-type": "application/json; charset=utf-8",
-    "date": "Tue, 11 Jan 2022 17:25:15 GMT",
-    "ms-cv": "pHBVJUbOxkW6ON/mX03xKA.0",
+    "date": "Wed, 02 Mar 2022 21:20:27 GMT",
+    "ms-cv": "owzk+pDQjEuVnbBCe6Qcnw.0",
     "request-context": "appId=",
-    "x-azure-ref": "0e73dYQAAAAD6/FN7jN4gT45wQANyU0NsU0FPMDNFREdFMDkwOQA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
+    "strict-transport-security": "max-age=2592000",
```

Comment:
Not sure if it's because of that reason, but there were failures using the prior recordings. Not all of them had to be recorded again, but I decided to run the whole test suite in record mode just to make sure everything works as expected.